mercapi/mercapi.py

```diff
@@ -22,19 +24,28 @@ class Mercapi:
 
     **Avoid instantiating this class more than once in a single runtime.**
     """
-
-    _user_agent = (
-        "Mozilla/5.0 (X11; Linux x86_64; rv:102.0) Gecko/20100101 Firefox/102.0"
-    )
-    _headers = {
-        "User-Agent": _user_agent,
-        "X-Platform": "web",
-    }
-
-    def __init__(self):
+    
+    def __init__(
+        self,
+        *,
+        proxies: typing.Optional[ProxiesTypes] = None,
+        user_agent: str = (
+            "Mozilla/5.0 (X11; Linux x86_64; rv:102.0) Gecko/20100101 Firefox/102.0"
+        ),
+    ):
+        """initialize
+
+        :param proxies: Proxy. ex) {"http://": "http://localhost:8080", "https://": "http://localhost:8080"}
```

Comment:
Thanks for taking care of documenting your changes, although I wouldn't mind if this were a bit more descriptive. Please let me know if you need help with it.

Reply:
I wrote a little more carefully.